SignalMessaging/attachments/MediaMessageView.swift

```diff
@@ -463,14 +472,8 @@ public class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
         guard let fromView = sender.view else {
             return
         }
-        guard let fromViewController = CurrentAppContext().frontmostViewController() else {
-            return
-        }
 
-        let window = CurrentAppContext().rootReferenceView
-        let convertedRect = fromView.convert(fromView.bounds, to:window)
-        let viewController = FullImageViewController(attachment:attachment, from:convertedRect)
-        viewController.present(from:fromViewController)
+        showMediaDetailViewController(fromView: fromView)
```

Comment:
We no longer present the full screen view controller (no called the MediaDetailViewController) directly, instead delegating to a protocol.